api4/user.go

```diff
@@ -42,6 +43,8 @@ func InitUser() {
 	BaseRoutes.User.Handle("/sessions", ApiSessionRequired(getSessions)).Methods("GET")
 	BaseRoutes.User.Handle("/sessions/revoke", ApiSessionRequired(revokeSession)).Methods("POST")
 	BaseRoutes.User.Handle("/audits", ApiSessionRequired(getAudits)).Methods("GET")
+
+	BaseRoutes.Users.Handle("/autocomplete/", ApiSessionRequired(autocompleteUsers)).Methods("GET")
```

Comment:
@jwilander actually that's a problem that I encounter. If i remove the / at the end, everything that comes after users/ will be interpreted as user_id, and thus generating an error ("Invalid or missing user_id parameter in request URL"). The only way I could force it to interpret /autocomplete as a route is adding another / at the end.